pkg/certificate/providers/tresor/fake/fake.go

```diff
@@ -25,7 +25,7 @@ func NewFake(msgBroker *messaging.Broker) *certificate.Manager {
 	if err != nil {
 		return nil
 	}
-	tresorCertManager, err := certificate.NewManager(ca, tresorClient, 1*time.Hour, msgBroker)
```

Comment:
let's keep this syntax for now, it will make subsequent PR's easier, and we can adapt as needed

Reply:
Reverted changes to the NewManager method's signature. The ca will not be used in NewManager, is that okay?